--- a/fcobjs.h
+++ b/fcobjs.h
@@ -51,7 +51,6 @@ FC_OBJECT (FOUNDRY,		FcTypeString,	FcCompareString)
 FC_OBJECT (ANTIALIAS,		FcTypeBool,	FcCompareBool)
 FC_OBJECT (HINT_STYLE,		FcTypeInteger,	FcCompareNumber)
 FC_OBJECT (HINTING,		FcTypeBool,	FcCompareBool)
-FC_OBJECT (CHAR_WIDTH,		FcTypeInteger,	NULL)
 FC_OBJECT (VERTICAL_LAYOUT,	FcTypeBool,	FcCompareBool)
 FC_OBJECT (AUTOHINT,		FcTypeBool,	FcCompareBool)
 FC_OBJECT (GLOBAL_ADVANCE,	FcTypeBool,	FcCompareBool)

